### PR TITLE
September and October SC updates

### PR DESCRIPTION
**Pull request by VeldaKiara**
Note updates for the Steering Council for the month of September and October

### CODE REVIEW

File: updates/2024-10-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+Core Developer Sprints:
+
+- Discussions about 2025 sprint location (Cambridge UK proposal)
+- Considerations for accessibility and travel logistics
+- Suggestion to create a document outlining sprint hosting requirements and costs
+
+Miscellaneous:
+
+- Office hours with another developer in residence scheduled for next week
+- [PEP 2026](https://peps.python.org/pep-2026/)to be discussed next week
```
Comment (JelleZijlstra) [inline updates/2024-10-steering-council-update.md:49]:
```suggestion
- [PEP 2026](https://peps.python.org/pep-2026/) to be discussed next week
```

File: updates/2024-09-steering-council-update.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+- Request for an informational PEP for PyREPL
+
+## 2024-09-18
+
+Core Developer Promotion:
+
+- The Steering Council unanimously accepts the community vote to promote Kirill Podoprigora to Core Developer
+- Steering Council to follow up on the promotion process
+
+Core Developer Sprint 2024:
```
Comment (hugovk) [inline updates/2024-09-steering-council-update.md:132]:
```suggestion
Core Developer Sprint 2025:
```

File: updates/2024-09-steering-council-update.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+- Considering ARM's proposal to host in Cambridge, UK
```
Comment (hugovk) [inline updates/2024-09-steering-council-update.md:134]:
```suggestion
- Considering Arm's proposal to host in Cambridge, UK
```

File: updates/2024-09-steering-council-update.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+- Pablo to discuss expectations and needs with Diego
+- Visa considerations to be clarified with ARM and PSF
```
Comment (hugovk) [inline updates/2024-09-steering-council-update.md:136]:
```suggestion
- Visa considerations to be clarified with Arm and PSF
```

File: updates/2024-10-steering-council-update.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+Release Management:
+
+- Preparations for next week's release underway
+- [Python 3.13 RC3](https://www.python.org/downloads/release/python-3130rc3/) released with Windows installer updates and the [Incremental GC rolled back](https://discuss.python.org/t/incremental-gc-and-pushing-back-the-3-13-0-release/65285)
+- Some concerns about [RC3](https://www.python.org/downloads/release/python-3130rc3/), mostly cosmetic issues to be addressed in 3.13.1
+- Performance discussions: mixed reports on speed compared to 3.12
+
+Core Developer Updates:
+
+- Someone nominated for triaging, open vote on discourse
```
Comment (hugovk) [inline updates/2024-10-steering-council-update.md:14]:
```suggestion
- Someone nominated for triaging, open vote on Discourse
```

File: updates/2024-10-steering-council-update.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+- Noted need for Python Software Foundation to have 4 weeks to implement new voting tools
+- Considered conducting mock elections to test new system
+
+Python 3.13 Release:
+
+- Announced release of [Python 3.13](https://docs.python.org/3/whatsnew/3.13.html)
+
+Core Developer Sprints:
+
+- Discussed potential ARM-hosted sprint in Cambridge UK
```
Comment (hugovk) [inline updates/2024-10-steering-council-update.md:83]:
```suggestion
- Discussed potential Arm-hosted sprint in Cambridge UK
```

File: updates/2024-10-steering-council-update.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+- Emphasized the importance of aligning grant work with larger project goals
+
+Core Developer Management:
+
+- Discussed process for removing commit bits from inactive core devs
+- Agreed to defer this discussion to the next Steering Council to avoid confusion with upcoming elections
+
+Core Developer Sprint:
+
+- Proceeding with ARM's offer to host in Cambridge UK
```
Comment (hugovk) [inline updates/2024-10-steering-council-update.md:113]:
```suggestion
- Proceeding with Arm's offer to host in Cambridge UK
```

File: updates/2024-09-steering-council-update.md (hunk 3 above)
Comment (gpshead) [reply updates/2024-09-steering-council-update.md:134]: heh, it's no longer an acryonym?  Acorn Risc Machines?  (applying)

**Review by Yhg1s — APPROVED** [review]

**Pull request merged**